IdentityServer/v7/docs/content/bff/overview.md

```diff
@@ -21,3 +21,58 @@ Browsers are increasingly restricting the use of cookies across site boundaries
 - The "silent renew" technique for session-bound token refreshing
 
 Using a BFF removes or mitigates all of these problems in the design. The backend component makes backchannel logout notifications possible, while still allowing the option of front-channel notifications for 1st party clients. Robust server-side session and token management with optional server-side sessions and refresh tokens take the place of OIDC Session Management and older token refresh mechanisms. As an ASP.NET Core server-side application, the BFF has access to a full featured and stable OpenID Connect client library that supports all the necessary protocol mechanisms and provides an excellent extensibility model for advanced features like [Mutual TLS]({{<ref "/tokens/pop/mtls" >}}), [DPoP]({{<ref "/tokens/pop/dpop" >}}), [JWT secured authorization requests]({{<ref "/tokens/jar">}}), and [JWT-based client authentication]({{<ref "/tokens/authentication/jwt">}}).
+
+## Logical and Physical Sessions
+
+When implemented correctly, a user will think of their time interacting with a solution as _"one session"_ also known as the **"logical session"**. The user should be unaware of the steps taken by developers to provide the experience and rightfully so. Users want to use the app, get their tasks completed, and log out happy. 
+
+{{<mermaid align="center">}}
+sequenceDiagram
+    actor Alice
+    Alice->>App: /login
+    App->>Alice: /account
+    box logical session
+        participant App
+    end
+{{< /mermaid >}}
+
+There is the user experience, and then there is the reality of the implementation, and for most distributed applications, including those implemented with BFF, **sessions are managed independently by each component of an application architecture.** 
```

Comment:
Too many clauses joined by commas. Suggest this sentence be shortened or divided.

Reply:
I think I re-worded this. @khalidabuhakmeh Do you agree with my rephrasing?